extensions/aws2-s3/runtime/pom.xml

```diff
@@ -59,6 +70,13 @@
             <groupId>software.amazon.awssdk.crt</groupId>
             <artifactId>aws-crt</artifactId>
```

Comment:
this should not be necessary, we provide graalvm substitutions https://github.com/quarkiverse/quarkus-amazon-services/blob/main/s3/runtime/src/main/java/io/quarkiverse/amazon/s3/runtime/S3CrtSubstitutions.java

Could be made available in a separate artifact for shared usage